lib/apiservers/portlayer/restapi/handlers/containers_handlers.go

```diff
@@ -118,6 +118,10 @@ func (handler *ContainersHandlersImpl) CreateHandler(params containers.CreatePar
 		Key:      pem.EncodeToMemory(&privateKeyBlock),
 		LayerID:  *params.CreateConfig.Image,
 		RepoName: *params.CreateConfig.RepoName,
+		Resources: executor.Resources{
+			NumCPUs: *params.CreateConfig.NumCpus,
+			Memory:  *params.CreateConfig.Memory,
```

Comment:
@jzt these go into the ConfigSpecConfig, not the executor config as they are an inherent part of the VM configuration: https://github.com/vmware/vic/blob/HEAD/lib/spec/spec.go#L57
Adding this into the executor would duplicate data and would not reflect out-of-band changes.